### DISCUSSION TITLE
Request for Permission to Use GitHub Keyword in my GitDown App

### DISCUSSION THREAD

**Discussion by KaziMahbuburRahman** [community/community#128221] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear GitHub Team,

I am writing to request permission to use certain elements of the GitHub brand in my mobile application, GitDown, which is currently under development/available on Google Play. The elements we wish to use include:

1. Use of the GitHub web embed in our app. So, that users can browse GitHub from our app and copy the repo link and download specific folder through our app.
2. Mention of the GitHub brand in the app name and description so that users can know that this app is a tool related to GitHub. For example: GitDown - GitHub Folder Downloader
3. Integration of GitHub-related functionality within the app. for example: using the API of GitHub to help users to download specific file/folder from GitHub.

GitDown aims to help to help GitHub users to download a specific folder rather than downloading or cloning the whole repository. We believe that using the GitHub brand elements will enhance the user experience.

We assure you that all usage will be in accordance with GitHub’s Trademark Policy and will clearly indicate that our app is not officially affiliated with or endorsed by GitHub.

We would be grateful if you could provide us with a signed letter granting us the necessary permissions. Please find attached additional details about our app for your reference.

Thank you for considering our request. We look forward to your positive response.

Best regards,
Kazi Mahbubur Rahman
Founder, GitDown

![GitDown-featured-image](https://github.com/community/community/assets/63844626/2910136d-86eb-4088-90e2-515a41f37007)

**Comment by ebndev** [comment, marked as answer]
Hi @NoobMahbub, thanks for being a part of the GitHub Community!

Please visit [this site ](https://github.com/logos)to view the terms for GitHub's Logo & Usage.

You will need to open a ticket on our [Support page](https://support.github.com/contact) if you need further permissions.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
